collections/business/general-notifications/event-general-notifications/account-statement-ready/account-statement-ready/push/en_US/engagement-template.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "../../../../../../../../schema/v1/push-content.json",
```

Comment:
@ex0b1t I don't think this is the correct approach to do and also we then will store this information on Content Services and might break the end to end flow. Btw we already have the Schema for the Engagement Template and it is here:

```
https://github.com/Backbase/engagements-data/blob/master/collections/retail/general-notifications/templates/account-balance-low-in-app-notification/content-schema.json
```